charts/timed/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: timed
-description: Chart for Timed application
+description: DEPRECATED use ghcr.io/adfinis/timed/helm/timed OCI Helm chart instead
 type: application
-version: 0.16.38
+version: 0.16.39
```

I'd say deprecation should potentially be an increase of the minor version, but other then that it's fine.
```suggestion
version: 0.17.0
```

---

it's not a feature and it doesn't matter so imma be lazy and just merge it as is ;)

i do agree that other cases may warrant doing a patch release for deprecation, in this case the way forward is to switch to the 4.x stream of timed anyway (and sticking with this chart is in fact a bug).